crates/ruff_python_formatter/src/statement/stmt_class_def.rs

```diff
@@ -33,6 +34,28 @@ impl FormatNodeRule<StmtClassDef> for FormatStmtClassDef {
 
         write!(f, [text("class"), space(), name.format()])?;
 
+        if !type_params.is_empty() {
```

Comment:
Does this mean we remove empty brackets (`class A[]: pass` -> `class A: pass`), and if so, is that intended?

Reply:
That is not valid syntax

```python
❯ python3.12 -c "class Foo[]: ..."
  File "<string>", line 1
    class Foo[]: ...
              ^
SyntaxError: invalid syntax
```

Reply:
That's good :)

Reply:
yep that definitely makes things easier, could you add a comment about that given that cases where empty optional parentheses aren't allowed are otherwise rare?

Reply:
👍